docs/how-to/create-slice.rst

```diff
@@ -0,0 +1,120 @@
+#. Search your package name through the list of slice definition files
+
+   * if you find it, open it and try to find a slice name containing the bits and pieces you need from that package
+
+
+Structure of a slice definitions file
+------------------------------------
+
+There is **one slice definitions file** for each Ubuntu package. All of the slice definitions files follow the same structure:
+
+.. code-block:: yaml
+
+    # (req) Name of the package.
+    # The slice definition file should be named accordingly (eg. "openssl.yaml")
+
+    package: <package-name>
+
+    # (req) List of slices
+    slices:
+
+        # (req) Name of the slice
+        <slice-name>:
+
+            # (opt) Optional list of slices that this slice depends on
+            essential:
+              - <pkgA_slice-name>
+              - ...
+
+            # (req) The list of files, from the package, that this slice will install
+            contents:
+                </path/to/content>:
+                </path/to/another/multiple*/content/**>:
+                </path/to/moved/content>: {copy: /bin/original}
+                </path/to/link>: {symlink: /bin/mybin}
+                </path/to/new/dir>: {make: true}
+                </path/to/file/with/text>: {text: "Some text"}
+                </path/to/mutable/file/with/default/text>: {text: FIXME, mutable: true}
+                </path/to/temporary/content>: {until: mutate}
+
+            # (opt) Mutation scripts, to allow for the reproduction of maintainer scripts,
+            # based on Starlark (https://github.com/canonical/starlark)
+            mutate: |
+                ...
+
+Find the dependencies of your package
+-------------------------------------
+
+The first step is to find the dependencies of the package for which you want to create a new slice definitions (``openssl`` in this guide). It's done like this:
+
+.. code-block:: sh
+
+    $ apt show openssl
+
+.. code-block:: text
+
+    package: openssl
+    Version: 3.0.2-0ubuntu1.7
+    Origin: Ubuntu
+    Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+    Original-Maintainer: Debian OpenSSL Team <pkg-openssl-devel@alioth-lists.debian.net>
+    Bugs: https://bugs.launchpad.net/ubuntu/+filebug
+    Depends: libc6 (>= 2.34), libssl3 (>= 3.0.2-0ubuntu1.2)
+
+From the above output, you can confirm that ``openssl`` **depends on** ``libc6`` and ``libssl3``. So when creating your slice definitions file for OpenSSL, you will need to remember to include those packages' slices as a dependency as well, whenever needed. Let's do that in the following section.
+
+Create your slice definition
+----------------------------
+
+You now have everything needed to actually define the OpenSSL slice that will install the content you are looking to have in the final ROCK.Since you are looking to install just the OpenSSL binaries from the ``openssl`` package, what about naming this new slice **bins**? Let's go for it:
```

Comment:
```suggestion
You now have everything needed to actually define the OpenSSL slice that will install the content you are looking to have in the final ROCK. Since you are looking to install just the OpenSSL binaries from the ``openssl`` package, what about naming this new slice **bins**? Let's go for it:
```